docs/policy.json.md

```diff
@@ -0,0 +1,248 @@
+<!-- NOTE: Keep this in sync with signature/policy_types.go! -->
+
+# Signature verification policy file format
+
+Signature verification policy files are used to specify policy, e.g. trusted keys,
+applicable when deciding whether to accept an image, or individual signatures of that image, as valid.
+
+The default policy is stored (unless overridden at compile-time) at `/etc/atomic/policy.json`;
```

Comment:
> is to use a set of standard rules for looking up config files. 

The basedir spec is essentially the same hard-coded path as above, + extra environment variable overrides; but it still requires the hard-coded path to be defined (and in typical use of the basedir spec, the environment variables are unset and the hard-coded path applies).

I would _love_ to use a platform-defined API for configuration storage (~ Windows registry or GConf2-like structured storage with documented schema), but I am not really aware of any in sufficiently wide use on Linux, especially outside the desktop space.

Still, even deferring to a third-party standard or configuration management API does not automatically resolve the question whether the signature verification policy “belongs” to the application or to the library — and my primary concern here is for all applications to use the same policy, i.e. for this repo to hard-code the location of the configuration within the configuration management API, whatever API is in use (where our API happens to be `ioutil.ReadFile("/etc/$something")`).